server/templates/details.html.tmpl

```diff
@@ -68,5 +68,8 @@
     <b class="font-bold">{{.Name}}</b>
     <span class="flex-none status-badge {{$s}}">{{$s | titlecase}}</span>
   </p>
-  <p class="text-dark-gray3 text-sm">{{or .Error .Description}}</p>
+  {{if (and .Description (ne .Status 0))}}
```

Comment:
I'm not sure if there's any way to reference the enum by name with Go templates, it's hard to understand that the 0 here means `StatusSkipped`.

Reply:
While I don't think you can reference the constants in the template, you could convert the `.Status` type to a string (e.g. the existing `$s` variable) and then compare the string representations, which might be clearer. The string representations are considered stable since we use them in other parts of the template.

```
{{if (and .Description (eq $s "skipped"))}}
```